pluto/docker-compose.yml

```diff
@@ -0,0 +1,95 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: pluto_frontend_1
+      APP_PORT: 7777
+    depends_on:
+      - frontend
+    restart: on-failure
+
+  discovery:
+    image: registry.gitlab.com/plutomining/pluto/pluto-discovery:0.6.0-rc.1@sha256:3da84654b98c07a64ffc93f14c525a742753f6afe776526de3adcf9c728169d0
+    network_mode: "host"
+    pid: host
+    cap_add:
+      - NET_RAW
+      - NET_ADMIN
+    restart: on-failure
+    environment:
+      - PORT=7775
+    volumes:
+      - ${APP_DATA_DIR}/data/leveldb:/home/node/app/data
+    user: "1000:1000"
+
+  backend:
+    image: registry.gitlab.com/plutomining/pluto/pluto-backend:0.6.0-rc.1@sha256:a49645cbb156ebb43ea4a36c024631cb0e26027b9324b0468ba31aa643055a25
+    pid: host
+    restart: on-failure
+    environment:
+      - PORT=7776
+      - AUTO_LISTEN=true
+      - DISCOVERY_SERVICE_HOST=http://172.17.0.1:7775
+      - GF_HOST=http://pluto_grafana_1:3000
```

Comment:
Fantastic work using the full container names for services here and elsewhere in this compose file 👌